### DISCUSSION TITLE
Is Trust Wallet considering Threshold Signature Scheme (TSS)?

### DISCUSSION THREAD

**Discussion by johnnynanjiang** [trustwallet/wallet-core#2642] [Ideas & Feature request]
I think private key / seed phrase management has been a pain point and bottleneck for massive adoption of crypto, probably a new paradigm is required for larger scale adoption.

**Comment by Milerius** [comment]
Right now it's not a priority, after reading https://academy.binance.com/en/articles/threshold-signatures-explained it also explains how works Threshold Wallet. Even if the UX/UI is an improvement for the user side, I do not think that's more secure in general.

TSS wallet also have vulnerabilities since they share seeds between 2 parties: https://eprint.iacr.org/2020/1052.pdf

I let the discussion open for debate, I do not think that TSS is ready yet for mass adoption.

**Reply by johnnynanjiang** [reply]
Thanks for the feedback and great reference @Milerius.

The vulnerabilities mentioned in `Attacking Threshold Wallets` seemed all fixed or fixable.

TSS might not be more secure compared to private key, somehow if it is more user friendly then I would reckon it would be a big win in terms of mass adoption.
